src/components/BaseLayout.tsx

```diff
@@ -7,6 +7,7 @@ interface Props {
   children: ReactNode;
   mainClassName?: string;
   contentClassName?: string;
+  dependencies?: DependencyList;
```

Comment:
We don't need the dependencies here, do we?

Reply:
no we don't, this should have been removed, will remove